README.md

```diff
@@ -248,65 +252,62 @@ func main() {
-Please see the tool's [README](https://github.com/hibiken/asynqmon) for details.
 
-Here's a few screenshots of the web UI.
+Here's a few screenshots of the Web UI:
 
 **Queues view**  
-![Web UI QueuesView](/docs/assets/asynqmon-queues-view.png)
-
-**Tasks view**  
-![Web UI TasksView](/docs/assets/asynqmon-task-view.png)
 
-## Command Line Tool
+![Web UI Queues View](https://user-images.githubusercontent.com/11155743/114697016-07327f00-9d26-11eb-808c-0ac841dc888e.png)
 
-Asynq ships with a command line tool to inspect the state of queues and tasks.
+**Tasks view**  
 
-Here's an example of running the `stats` command.
+![Web UI TasksView](https://user-images.githubusercontent.com/11155743/114697070-1f0a0300-9d26-11eb-855c-d3ec263865b7.png)
 
-![Gif](/docs/assets/demo.gif)
+**Settings and adaptive dark mode**
 
-For details on how to use the tool, refer to the tool's [README](/tools/asynq/README.md).
+![Web UI Settings and adaptive dark mode](https://user-images.githubusercontent.com/11155743/114697149-3517c380-9d26-11eb-9f7a-ae2dd00aad5b.png)
 
-## Installation
+For details on how to use the tool, refer to the tool's [README](https://github.com/hibiken/asynqmon#readme). 
 
-To install `asynq` library, run the following command:
+## Command Line Tool
 
-```sh
-go get -u github.com/hibiken/asynq
-```
+Asynq ships with a command line tool to inspect the state of queues and tasks.
 
 To install the CLI tool, run the following command:
 
 ```sh
 go get -u github.com/hibiken/asynq/tools/asynq
 ```
 
-## Requirements
+Here's an example of running the `asynq stats` command:
+
+![Gif](/docs/assets/demo.gif)
 
-| Dependency                 | Version |
-| -------------------------- | ------- |
-| [Redis](https://redis.io/) | v3.0+   |
-| [Go](https://golang.org/)  | v1.13+  |
+For details on how to use the tool, refer to the tool's [README](/tools/asynq/README.md).
 
 ## Contributing
 
-We are open to, and grateful for, any contributions (Github issues/pull-requests, feedback on Gitter channel, etc) made by the community.
+We are open to, and grateful for, any contributions (GitHub issues/PRs, feedback on [Gitter channel](https://gitter.im/go-asynq/community), etc) made by the community.
+
 Please see the [Contribution Guide](/CONTRIBUTING.md) before contributing.
 
 ## Acknowledgements
 
-- [Sidekiq](https://github.com/mperham/sidekiq) : Many of the design ideas are taken from sidekiq and its Web UI
-- [RQ](https://github.com/rq/rq) : Client APIs are inspired by rq library.
-- [Cobra](https://github.com/spf13/cobra) : Asynq CLI is built with cobra
+- [Sidekiq](https://github.com/mperham/sidekiq): Many of the design ideas are taken from sidekiq and its Web UI
+- [RQ](https://github.com/rq/rq): Client APIs are inspired by rq library.
+- [Cobra](https://github.com/spf13/cobra): Asynq CLI is built with cobra
+
+## Stargazers
+
+![Stargazers Asynq](https://starchart.cc/hibiken/asynq.svg)
```

Comment:
Can we remove the "Acknowledgements" and "Stargazers" sections?

Reply:
Okay. No problem!